packages/back-end/src/events/handlers/webhooks/event-webhooks-utils.ts

```diff
@@ -51,114 +36,3 @@ export const getEventWebHookSignatureForPayload = <T>({
 };
 
 // endregion Web hook signing
-
-// region Web hook Payload creation
-
-type BasePayloadCreatorOptions = {
-  organization: OrganizationInterface;
-  savedGroupMap: GroupMap;
-};
-
-export const getPayloadForNotificationEvent = ({
-  event,
-  organization,
-  savedGroupMap,
-}: BasePayloadCreatorOptions & {
-  event: NotificationEvent;
-}): NotificationEventPayload<
-  NotificationEventName,
-  NotificationEventResource,
-  unknown
-> | null => {
-  switch (event.event) {
-    case "experiment.created":
-    case "experiment.updated":
-    case "experiment.deleted":
-      // TODO: https://linear.app/growthbook/issue/GB-18
-      return null;
-
-    case "feature.created":
-      return getPayloadForFeatureCreated({
-        event,
-        organization,
-        savedGroupMap,
-      });
-    case "feature.updated":
-      return getPayloadForFeatureUpdated({
-        event,
-        organization,
-        savedGroupMap,
-      });
-    case "feature.deleted":
-      return getPayloadForFeatureDeleted({
-        event,
-        organization,
-        savedGroupMap,
-      });
-  }
-};
-
-const getPayloadForFeatureCreated = ({
-  event,
-  organization,
-  savedGroupMap,
-}: BasePayloadCreatorOptions & {
-  event: FeatureCreatedNotificationEvent;
-}): NotificationEventPayload<
-  "feature.created",
-  "feature",
-  { feature: ApiFeature }
-> => ({
-  ...event,
-  data: {
-    ...event.data,
-    feature: getApiFeatureObj(event.data.current, organization, savedGroupMap),
```

Comment:
Removing this payload creator changes it from `feature: ApiFeature` to `current: ApiFeature` which is more consistent with how the events are logged. This will cause a breaking change to anyone using this, but the feature is in beta. This impacts only the `feature.created` event. The other events were already using `current` and `previous` keys.